### ISSUE TITLE
Ability to turn off analytics-node

### ISSUE THREAD

**Issue by wlingke** [segmentio/analytics-node#116]
Because analytics-node is part of our controllers, it would be nice to have the ability to turn it off when we are running our unit tests by passing in an option like:

```
var analytics = new Analytics('YOUR_WRITE_KEY', { off: true });
```

**Comment by vadimdemedes** [comment]
I think this is a valid use case. I'd name this option simply `enable`, rather than `off`.

**Comment by wlingke** [comment]
Awesome! Looking forward to seeing it

**Comment by wlingke** [comment]
@vadimdemedes Hey there - any updates on this??

Thanks!

**Comment by f2prateek** [comment]
@wlingke sorry for the delay, we're about 90% through with the PR but needs some changes still. we'll try get this done soon. Tracking internally at https://segment.atlassian.net/browse/LIB-118

**Comment by buskerone** [comment]
I think this is super important to have when running unit tests. Any updates?

**Comment by vadimdemedes** [comment]
@wlingke @buskerone Would either of you be interested in making a PR for it? https://github.com/segmentio/analytics-node/issues/116#issuecomment-318745025 should be a simple feature.

**Comment by buskerone** [comment]
@vadimdemedes sure if I have enough time I'll do it but for this repo because that's what I'm using right now -> https://github.com/neiker/analytics-react-native 
I'll update u

**Comment by wlingke** [comment]
I'll throw in a PR this weekend. Wish you would've let me known sooner. It
sounded like the segment team was 90% done with this 3 mo ago.

**Comment by vadimdemedes** [comment]
@wlingke You're right, it's our bad, it was 90% done and then I moved from working on this project to internal stuff and abandoned my PR.

**Comment by wlingke** [comment]
Do you still have your pr? If not happy to start from scratch

**Comment by vadimdemedes** [comment]
Yes, here it is: https://github.com/segmentio/analytics-node/pull/118.

**Comment by wlingke** [comment]
Okay thanks. Any particular reason why this didn't get merged in? It looks
good for the most part -- except preventing modification of enabled.

**Comment by vadimdemedes** [comment]
Yep, that's the only reason.

**Comment by wlingke** [comment]
Cool that's easy. I'll get a PR up this weekend.

**Comment by wlingke** [comment]
@vadimdemedes PR made #158

**Comment by vadimdemedes** [comment]
Fixed via #158.
